x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/rules_table/guided_onboarding/translations.ts

```diff
@@ -10,14 +10,15 @@ import { i18n } from '@kbn/i18n';
 export const INSTALL_PREBUILT_RULES_TITLE = i18n.translate(
   'xpack.securitySolution.detectionEngine.rules.guidedOnboarding.installPrebuiltRules.title',
   {
-    defaultMessage: 'Load the Elastic prebuilt rules',
+    defaultMessage: 'Install your first prebuilt rule',
   }
 );
 
 export const INSTALL_PREBUILT_RULES_CONTENT = i18n.translate(
   'xpack.securitySolution.detectionEngine.rules.guidedOnboarding.installPrebuiltRules.content',
   {
-    defaultMessage: 'To get started you need to load the Elastic prebuilt rules.',
+    defaultMessage:
+      'Navigate to the Add Elastic Rules page and install the example "My First Rule".',
```

Comment:
Hardcoding the name of the rule here is not ideal. But the alternative is calling the `_review` endpoint to fetch all available rules. Even in that case, the endpoint doesn't allow serverside filtering as of yet, so we'd have to retrieve all available rules and then filter in the frontend according to `name` or `ruleId`, which would need to be hardcoded as well.